include/sentry.h

```diff
@@ -1409,9 +1409,13 @@ SENTRY_API sentry_uuid_t sentry_capture_event(sentry_value_t event);
  * This generates a fatal error event, includes the scope and attachments.
  * If the event isn't dropped by a before-send hook, the minidump is attached
  * and the event is sent.
+ * 
+ * If returns a nil UUID, the minidump was not successfully captured and sent to transport (not backend yet).
+ * Otherwise, will return the UUID of the event sent to transport with the minidump attachment.
+ * This can be used look up the upload later.
```

Comment:
```suggestion
 * Returns a nil `UUID` if capturing the minidump failed and the event-id
 * otherwise. Uploads can fail because capturing is asynchronous, so a non-nil
 * `UUID` is not a delivery guarantee. However, if the minidump is successfully
 * delivered, the ID is guaranteed to be the same as the event in the Sentry UI.```